app/controllers/api/v2/hostgroups_controller.rb

```diff
@@ -94,12 +94,32 @@ def clone
         process_response @hostgroup.save
       end
 
+      api :PUT, "/hostgroups/:id/rebuild_config", N_("Rebuild orchestration config")
+      param :id, :identifier, :required => true
+      param :only, Array, :desc => N_("Limit rebuild steps, valid steps are %{host_rebuild_steps}")
```

Comment:
When does `%{host_rebuild_steps}"` get replaced by the real values?

Reply:
@iNecas I think this is set in config/initializers/apipie.rb:
 https://github.com/theforeman/foreman/blob/develop/config/initializers/apipie.rb#L25

Reply:
You're right, thanks for pointers